### DISCUSSION TITLE
VS2017 编译最新报错

### DISCUSSION THREAD

**Discussion by sig999** [CefView/QCefView#114] [Q&A] [answered]
QT 5.10 vc140 设置，
cef_bind_internal.h 1249 行#无法从“R (__cdecl *)(base::internal::BindStateBase *,conditional<_Test,const std::string&,const std::string&>::type)”转换##为“PolymorphicInvoke”

**Comment by tishion** [comment]
你自己解决吧，我没有这么多环境。

**Comment by sig999** [comment, marked as answer]
cef_binary_95.7.12+g99c4ac0 需要用VS 2019才能编译过，因为有的是c14语法，另外QT 工程低版本的QMetaObject::invokeMethod 需要改成低版本写法，另外QCefView 有一些简单的语法错误

**Comment by doasyourself** [comment]
编译不过，至少解释一下吧

**Reply by tishion** [reply]
至少说一下你遇到的问题？：）

**Reply by gengshenchen** [reply]
大佬能帮回答下吗，需要设置socks5 代理这些，QCefView没找到哪里可以配置，是需要使用cef吗

**Reply by tishion** [reply]
config.addCommandLineSwitchWithValue("proxy-server", "socks5://127.0.0.1:8888");

**Reply by gengshenchen** [reply]
谢谢，是我懵le
